apps/openchallenges/challenge-service/src/main/resources/db/migration/V1.0.1__insert_temp_data.sql

```diff
@@ -2468,645 +2468,687 @@ VALUES ('1', 1, 75, 'sponsor'),
-  ('566', 103, 189, 'challenge_organizer'),
-  ('567', 103, 34, 'challenge_organizer'),
-  ('568', 103, 62, 'challenge_organizer'),
-  ('569', 103, 131, 'challenge_organizer'),
-  ('570', 103, 177, 'challenge_organizer'),
-  ('571', 103, 189, 'sponsor'),
-  ('572', 104, 12, 'challenge_organizer'),
-  ('573', 104, 215, 'challenge_organizer'),
-  ('574', 104, 84, 'challenge_organizer'),
-  ('575', 104, 7, 'challenge_organizer'),
-  ('576', 104, 15, 'sponsor'),
-  ('577', 104, 90, 'sponsor'),
-  ('578', 104, 103, 'sponsor'),
-  ('579', 104, 50, 'sponsor'),
-  ('580', 105, 12, 'challenge_organizer'),
-  ('581', 105, 147, 'sponsor'),
-  ('582', 105, 35, 'sponsor'),
-  ('583', 106, 12, 'challenge_organizer'),
-  ('584', 106, 186, 'sponsor'),
-  ('585', 106, 178, 'sponsor'),
-  ('586', 107, 12, 'challenge_organizer'),
-  ('587', 107, 77, 'sponsor'),
-  ('588', 124, 12, 'challenge_organizer'),
-  ('589', 124, 215, 'challenge_organizer'),
-  ('590', 124, 84, 'challenge_organizer'),
-  ('591', 124, 7, 'challenge_organizer'),
-  ('592', 124, 98, 'sponsor'),
-  ('593', 124, 139, 'sponsor'),
-  ('594', 124, 162, 'sponsor'),
-  ('595', 161, 13, 'challenge_organizer'),
-  ('596', 161, 237, 'challenge_organizer'),
-  ('597', 162, 13, 'challenge_organizer'),
-  ('598', 162, 238, 'challenge_organizer'),
-  ('599', 163, 13, 'challenge_organizer'),
-  ('600', 163, 238, 'challenge_organizer'),
-  ('601', 164, 13, 'challenge_organizer'),
-  ('602', 164, 231, 'challenge_organizer'),
-  ('603', 165, 13, 'sponsor'),
-  ('604', 55, 1, 'sponsor'),
-  ('605', 53, 1, 'sponsor'),
-  ('606', 52, 1, 'sponsor'),
-  ('607', 54, 1, 'sponsor'),
-  ('608', 166, 1, 'sponsor'),
-  ('609', 45, 1, 'sponsor'),
-  ('610', 44, 1, 'sponsor'),
-  ('611', 43, 1, 'sponsor'),
-  ('612', 40, 1, 'sponsor'),
-  ('613', 42, 1, 'sponsor'),
-  ('614', 39, 1, 'sponsor'),
-  ('615', 41, 1, 'sponsor'),
-  ('616', 35, 1, 'sponsor'),
-  ('617', 36, 1, 'sponsor'),
-  ('618', 34, 1, 'sponsor'),
-  ('619', 37, 1, 'sponsor'),
-  ('620', 33, 1, 'sponsor'),
-  ('621', 32, 1, 'sponsor'),
-  ('622', 31, 1, 'sponsor'),
-  ('623', 30, 1, 'sponsor'),
-  ('624', 29, 1, 'sponsor'),
-  ('625', 21, 1, 'sponsor'),
-  ('626', 27, 1, 'sponsor'),
-  ('627', 26, 1, 'sponsor'),
-  ('628', 28, 1, 'sponsor'),
-  ('629', 24, 1, 'sponsor'),
-  ('630', 25, 1, 'sponsor'),
-  ('631', 23, 1, 'sponsor'),
-  ('632', 20, 1, 'sponsor'),
-  ('633', 19, 1, 'sponsor'),
-  ('634', 18, 1, 'sponsor'),
-  ('635', 17, 1, 'sponsor'),
-  ('636', 16, 1, 'sponsor'),
-  ('637', 15, 1, 'sponsor'),
-  ('638', 14, 1, 'sponsor'),
-  ('639', 8, 1, 'sponsor'),
-  ('640', 13, 1, 'sponsor'),
-  ('641', 7, 1, 'sponsor'),
-  ('642', 11, 1, 'sponsor'),
-  ('643', 12, 1, 'sponsor'),
-  ('644', 10, 1, 'sponsor'),
-  ('645', 9, 1, 'sponsor'),
-  ('646', 6, 1, 'sponsor'),
-  ('647', 7, 1, 'sponsor'),
-  ('648', 5, 1, 'sponsor'),
-  ('649', 4, 1, 'sponsor'),
-  ('650', 1, 1, 'sponsor'),
-  ('651', 3, 1, 'sponsor');
+  ('13', 5, 134, 'challenge_organizer'),
```

Comment:
The property `id` of the table `challenge_contribution` is defined as an int, but here data set the ids as string.

Reply:
I thought that was weird too, but I just followed suit what was [done previously](https://github.com/Sage-Bionetworks/sage-monorepo/blob/027f56280792878183549532dd55d2a13bcbf69d/apps/openchallenges/challenge-service/src/main/resources/db/migration/V1.1.0__insert_data.sql).